stdlib/hashtbl.ml

```diff
@@ -374,24 +403,35 @@ module type S =
   sig
     type key
     type 'a t
-    val create: int -> 'a t
+    val create : int -> 'a t
```

Comment:
Spurious whitespace additions (and the most common style is rather to drop such spaces).